resources/private_key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Uploading a private key feels like "I'm doing it wrong", but I'm not sure how else to be able to run the ssh key tests locally? Is there a way to generate this each time the tests are run and use that instead?

Reply:
This is probably ok otherwise you'd have to generate a key when bringing up the containers.
It should go in fixtures though.